website/static/js/fangorn.js

```diff
@@ -38,6 +38,23 @@ $.extend(EXTENSION_MAP, {
     gsheet: 'xlsx'
 });
 
+var _defaultIconState = function () {
+    return {
+        mode : 'bar',
+        generalIcons : {
+            search : { on : true, template : searchIcon },
```

Comment:
No they are atttached to 
```js
treebeard.options.iconState.generalIcons
```

the default state is a method to not manually replicate the initial state so there is a function that returns it, but theoretically it can be placed in other places.